frontend/app/lib/server/open-payments.server.ts

```diff
@@ -0,0 +1,319 @@
+import {
+  type AuthenticatedClient,
+  type PendingGrant,
+  type Quote,
+  type WalletAddress,
+  createAuthenticatedClient,
+  isFinalizedGrant,
+  isPendingGrant
+} from '@interledger/open-payments'
+import { createId } from '@paralleldrive/cuid2'
+import { toWalletAddressUrl } from '../utils.js'
+
+async function createClient() {
+  return await createAuthenticatedClient({
+    keyId: process.env.OP_KEY_ID!,
+    privateKey: Buffer.from(process.env.OP_PRIVATE_KEY!, 'base64'),
+    walletAddressUrl: process.env.OP_WALLET_ADDRESS!
+  })
+}
```

Comment:
Can cache this client.

```suggestion
let client
async function createClient() {
  client ??= await createAuthenticatedClient({
    keyId: process.env.OP_KEY_ID!,
    privateKey: Buffer.from(process.env.OP_PRIVATE_KEY!, 'base64'),
    walletAddressUrl: process.env.OP_WALLET_ADDRESS!
  })
  return client
}
```

Reply:
this fails typecheck, so I reverted it
the original function is from other implementations, it can live as is for now